cccm-frontend/src/router/index.ts

```diff
@@ -59,22 +58,13 @@ const router = new VueRouter({
       }
     },
     {
-      path: '/clientrecord/:csNumber/crnacmp/:formID',
-      name: 'crnacmp',
-      component: CrnaCmpView,
+      path: '/clientrecord/:csNumber/form/:formType/:formID/:linkedSara',
```

Comment:
Is there a cccm endpoint for this or is this just a placeholder for now?

Reply:
this is the normalized URL to access 'CRNA' or 'SARA' form